plan_review_checklist.md

```diff
@@ -0,0 +1,61 @@
+
+Have a committer representative open a new Spec. repository PR for the specification version and paste the following check-list into the PR conversation (comments).
+
+----
+1. Spec PR for Plan / Proposal
+  - [ ] PR uses [template](https://github.com/jakartaee/specifications/blob/master/pull_request_template.md)
+  - [ ] Directory of form {spec}/x.y
+  - [ ] (Optional) PDF of form jakarta-{spec}-spec-x.y.pdf ("-spec" preferred but not required)
+  - [ ] (Optional) HTML of form jakarta-{spec}-spec-x.y.html ("-spec" preferred but not required)
+  - [ ] Index page {spec}/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
+  - [ ] Index page {spec}/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_index_template.md)
+  - [ ] No other files (e.g., no jakarta_ee_logo_schooner_color_stacked_default.png)
+
+2. _index.md
+  - [ ] Link to project PMI plan of the form https://projects.eclipse.org/projects/ee4j.{spec}/releases/x.y/plan
+  - [ ] Scope statement changes -- This might be any of
+    * Revised specification document, possibly with markup indicating changes
+    * Short description of proposed changes
+    * List of issues 
+        * If many issues are involved consider using an Epoch type issue (issue of issues) and only provide high-level summary. 
+    * If the version update includes a material that are outside the original scope, (i.e. if a feature were to be refactored to another specification or if some major additional functionality is proposed), this should be highlighted.
+  - [ ] High level Plan -- If intended to align with a more broad release (i.e. Jakarta EE major release) reference that plan (or plan body). Indicate the milestones intended for *this* specification update (Any planned checkpoints, Feature Complete, Code Complete, TCK Complete, Target Ballot date). 
+  - [ ] How to communicate with the Specification committer team (team list, Issue tracker, etc.)
+  - [ ] Any additional requirements or details that would be necessary to complete this specification (dependencies, resource requirements, etc.)
+
+3. Proposed Spec PDF (Optional)
+  - [ ] Correct spec title, clearly marked preliminary
+  - [ ] Correct Eclipse copyright line
+  - [ ] Must indicate DRAFT or SNAPSHOT
+  - [ ] Correct Logo
+
+4. Proposed Spec HTML (Optional)
+  - [ ] Same as PDF
+
+5. TCK
+  - [ ] Brief outline describing how TCK will be developed.
+  - [ ] Name or list of target implementation partners
+
+6. EMO Requirements 
+    TBD
+    - [ ] An issue will be created by the EMO to track the release review.
+----
+
+## Support and Guidance Resources (Informative)
+The mentor should remind the specification committer team of the various process requirements and also the support that is available from the various committees. This can be done via e-mail or a face to face meeting as appropriate.
```

Comment:
```suggestion
The mentor should remind the specification committer team of the various process requirements and also the support that is available from the various committees. This can be done via e-mail or a face-to-face meeting as appropriate.
```